src/utils.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { getLocalizedString } from './utils';
+
+describe(__filename, () => {
+  describe('getLocalizedString', () => {
```

Comment:
Would it be worth adding a test with multiple languages in the map, to verify that we are returning the *correct* one?

Reply:
Maybe, although it is a map so there is no risk of collision.

Reply:
I added a second item in one of the maps.